backend/app/Http/Controllers/UserController.php

```diff
@@ -36,4 +36,12 @@ public function login(Request $request)
 
         return LoginResource::make(array_merge($user->toArray(), ['token' => $token]));
     }
+    public function myAccounts(Request $request)
+    {
+        $user_id = $request->user_id;
+
+        $user = User::findOrFail($user_id);
```

Comment:
current logged in user can be easily retrieved by
Auth::user(),

then for id Auth::id()